xml/System.Formats.Cbor/CborConformanceMode.xml

```diff
@@ -34,7 +34,15 @@
       </ReturnValue>
       <MemberValue>2</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Ensures that the CBOR data is canonical, as specified in RFC7049 section 3.9.
```

Comment:
It would be nice to have hyperlinks to the specs mentioned.

Reply:
Agreed, but this shouldn't be a merge blocker. If anyone is interested in reading the specs, they can simply search for them on the internet.